developerportal/apps/people/templates/people.html

```diff
@@ -6,7 +6,7 @@
 {% block body_class %}people{% endblock %}
 
 {% block content %}
-  <main role="main">
```

Comment:
Ah! I see. The validator is marking this with a warning as technically, in supporting browsers, the `role` is not required:

```
Warning: The main role is unnecessary for element main.
From line 7, column 1; to line 7, column 18
```

Wonder if one could suppress that specific warning? Any effort on this would need to be measured against the actual number of users we have using browsers that do not support the `main` element and the resulting landmark role.

Reply:
Given this is all being prepped on an epic branch, I'll ticket that as a separate thing. Thanks for the feedback!